hack/windows/InstallOVS.ps1

```diff
@@ -0,0 +1,131 @@
+Param(
```

Comment:
If this file will be provided for downloading, should we put it under build/?
@antoninbas

Reply:
I asked Rui to move this script here. IMO this is a convenience script that some users may choose not to use at all. In any case, it has to be run "manually", it is not something that Antrea runs for you. I think build/ is more about the docker images, and the scripts we package into them.

Reply:
Ok.